docs/guides/testing.md

```diff
@@ -26,6 +26,9 @@ gulp protractor
 ```
 **Hint**: `gulp watch` and `gulp protractor` should not be run at the same time, this will yield unexpected and weird results. Make sure to run them separately!
 
+**Advanced**: You can start both commands with optional parameter `--conf=configfile.js`. So in case you have different setup configurations, you can run
```

Comment:
I'd write `[...] with **an** optional parameter [...]` and put a semicolon here: `[...] you can run:`